### PR TITLE
Updated GitHub Action workflows

### LINKED ISSUE #420: Update ansible-mso CI pipeline
Update GitHub actions and resolve issues with Galaxy Importer.

### PR DESCRIPTION
**Pull request by samiib**
- Updated ansible versions
- Updated action versions
- Fixed some yaml formatting issues
- Uncommented Unit tests
Fixes #420

### CODE REVIEW

File: .github/workflows/galaxy-importer.cfg [hunk 1]
```diff
@@ -6,4 +6,5 @@ RUN_ANSIBLE_LINT = True
 RUN_ANSIBLE_TEST = False
 ANSIBLE_TEST_LOCAL_IMAGE = False
 LOCAL_IMAGE_DOCKER = False
-INFRA_OSD = False
\ No newline at end of file
+INFRA_OSD = False
+CHECK_CHANGELOG = False
```
Comment (akinross) [inline .github/workflows/galaxy-importer.cfg:10]: Is there a reason we do not do this `CHECK_CHANGELOG` ansible aci?

Comment (samiib) [reply .github/workflows/galaxy-importer.cfg:10]: The changelog file had incorrect indenting in MSO but in ND and ACI the indenting was fine. 🤔 
I thought this setting would ignore it but it did not. I will revert this line.

Comment (samiib) [reply .github/workflows/galaxy-importer.cfg]: Done

**Review by gmicol — APPROVED** [review]
LGTM, same comment as for ansible-nd.

**Review by akinross — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**